cli/main.go

```diff
@@ -1228,10 +1226,18 @@ func ValidateOutput(seed *objects.Seed, outDir string) {
 			fmt.Fprintf(os.Stderr, "ERROR: %s is invalid: - %s\n", constants.ResultsFileManifestName, desc)
 		}
 	}
+	
+	matches, _ := filepath.Glob(path.Join(outDir, "*.metadata.json"))
```

Comment:
Let's look at just checking for sidecar files that are associated to an already matched output files.